src/PlotRecipes/ElectricField.jl

```diff
@@ -38,43 +38,97 @@ end
     ElectricPotential(ef_magn,g), cross_section, idx, value, contours_equal_potential
 end
 
+function get_sample_lines(dim_symbol::Symbol, v::T, grid::Grid{T,3,Cartesian}, sampling::T)::Vector{Line} where {T}
+    
+    xrange = range(round.(endpoints(grid.x.interval)./sampling, (RoundDown, RoundUp)).*sampling..., step = sampling)
+    yrange = range(round.(endpoints(grid.y.interval)./sampling, (RoundDown, RoundUp)).*sampling..., step = sampling)
+    zrange = range(round.(endpoints(grid.z.interval)./sampling, (RoundDown, RoundUp)).*sampling..., step = sampling)
+    
+    return if dim_symbol == :x
+        vcat(
+            [Line(CartesianPoint{T}(v,y,0), CartesianVector{T}(0,0,1)) for y in yrange],
+            [Line(CartesianPoint{T}(v,0,z), CartesianVector{T}(0,1,0)) for z in zrange]
+        )
+    elseif dim_symbol == :y
+        vcat(
+            [Line(CartesianPoint{T}(x,v,0), CartesianVector{T}(0,0,1)) for x in xrange],
+            [Line(CartesianPoint{T}(0,v,z), CartesianVector{T}(1,0,0)) for z in zrange]
+        )
+        
+    else # dim_symbol == :z
+        vcat(
+            [Line(CartesianPoint{T}(x,0,v), CartesianVector{T}(0,1,0)) for x in xrange],
+            [Line(CartesianPoint{T}(0,y,v), CartesianVector{T}(1,0,0)) for y in yrange]
+        )
+    end
+end
+
+
+function get_sample_lines(dim_symbol::Symbol, v::T, grid::Grid{T,3,Cylindrical}, sampling::T)::Vector{Line} where {T}
+    
+    rrange = range(round.((-grid.r.interval.right,grid.r.interval.right)./sampling, (RoundDown, RoundUp)).*sampling..., step = sampling)
+    #φrange = range(round.(endpoints(grid.φ.interval)./sampling, (RoundDown, RoundUp)).*sampling..., step = sampling)
+    zrange = range(round.(endpoints(grid.z.interval)./sampling, (RoundDown, RoundUp)).*sampling..., step = sampling)
+    
+    
+    return if dim_symbol == :φ
+        vcat(
+            [Line(CartesianPoint{T}(r*cos(v),r*sin(v),0), CartesianVector{T}(0,0,1)) for r in rrange],
+            [Line(CartesianPoint{T}(0,0,z), CartesianVector{T}(cos(v),sin(v),0)) for z in zrange]
+        )
+    #=
+    elseif dim_symbol == :z
+        vcat(
+            [Line(CartesianPoint{T}(x,0,v), CartesianVector{T}(0,1,0)) for x in xrange],
+            [Line(CartesianPoint{T}(0,y,v), CartesianVector{T}(1,0,0)) for y in yrange]
+        )
+    =#
```

Comment:
Can we just take the same as for the Cartesian grid?
We just take -r_max:+r_max as ranges for `y` and `x`.

Reply:
This changed with my latest commit (d559dac).

Reply:
Ah, alright :+1: